sdk/openfeature-nodejs-provider/src/DevCycleProvider.ts

```diff
@@ -165,11 +178,22 @@ export default class DevCycleProvider implements Provider {
     private resultFromDVCVariable<T>(
         variable: DVCVariable,
     ): ResolutionDetails<T> {
+        const flagMetadata: FlagMetadata = {
```

Comment:
@toddbaert / @beeme1mr is this the kind of data you were expecting to be included in `flagMetadata`?

Reply:
The data returned in `flagMetadata` was intended to be used for debugging and telemetry. I'm not sure how valuable returning information about defaults is but it's completely up to you. Perhaps instead of returning `isDefaulted` you could set the [resolution detail](https://openfeature.dev/specification/types) to `DEFAULT`.

I'm not sure if you have access to this information in the SDK but including things like the organization, project, and environment would be very useful.

Reply:
We already set the resolution detail, I guess these aren't needed then. We do have access to org / project / env. however, those aren't publically in our SDK, we will need to think about how to do that.